Sources/FINNSetup/BackendModel/FilterSetup.swift

```diff
@@ -81,7 +81,7 @@ public struct FilterSetup: Decodable {
             return Filter.search(key: key)
         case FilterKey.preferences.rawValue:
             let subfilters = config.preferenceFilters.compactMap {
-                filterData(forKey: $0).map({ makeListFilter(from: $0, withStyle: .normal) })
+                filterData(forKey: $0).flatMap({ makeListFilter(from: $0, withStyle: .normal) })
```

Comment:
Is this suppose to be here?

Reply:
Yes, because `makeListFilter` returns optional now.